templates/zombienet/Cargo.toml

```diff
@@ -11,10 +11,10 @@ publish = false
 
 [dependencies]
 env_logger = "0.11.2"
-log = "0.4"
-tokio = { version = "1.36.0", features = ["rt-multi-thread"] }
-anyhow = "1.0.81"
-zombienet-sdk = "0.2.8"
+log = { workspace = true }
+tokio = { workspace = true, features = ["rt-multi-thread"] }
+anyhow = { workspace = true }
+zombienet-sdk = "0.2.13"
```

Comment:
Shouldn't we put this one in the workspace as well ?

Reply:
Yes, I will move it 👍 .